### PR TITLE
Python3 safety

### PR DESCRIPTION
**Pull request by lgritz**
Fix the things flagged by pylint --py3k

Fixes #1045

### CODE REVIEW

**Comment by lgritz** [comment]
Fixes #1068

**Comment by lgritz** [comment]
Not 1045, which, oops, was another attempt at fixing this that I'd forgotten about. But I think I'll stick to this one.

File: src/liboslexec/serialize-bc.py [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 f_out = open(out_name, 'w')
 f_out.write('#include <cstddef>\n')
-f_out.write('unsigned char %s_block[] = {\n' % prefix)
+f_out.write('unsigned char ' + prefix + '_block[] = {\n')
 f_in.read
-for c in f_in.read():
-    f_out.write('0x%s,\n' % c.encode('hex'))
+if (sys.version_info > (3, 0)):
+    for c in f_in.read():
+        f_out.write(hex(c) + '\n')
```
Comment (sambler) [inline src/liboslexec/serialize-bc.py]: You missed the comma after each value

f_out.write(hex(c) + ',\n')

Comment (lgritz) [reply src/liboslexec/serialize-bc.py]: thanks for the tip. updated.

**Comment by lgritz** [comment]
Seems to work. Merging.

**Pull request merged**